### DISCUSSION TITLE
We need contributor license agreements

### DISCUSSION THREAD

**Discussion by jankapunkt** [openqda/openqda#141] [Ideas]
Problem: we need to make sure, external contributors will not make claims in the future in terms of code ownership
Solution: contributor license agreement
Good thing: It can be automated: https://cla-assistant.io/

**Comment by kadewe** [comment]
ok, so are you going to create a "standard contributor license agreement" for OpenQDA? Where can I read more about the implications?

**Reply by jankapunkt** [reply]
Actually the wikipedia article is indeed a good starting point https://en.wikipedia.org/wiki/Contributor_License_Agreement
